subiquity/common/geoip.py

```diff
@@ -14,19 +14,53 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
I failed to notice this last time, but the idea behind subiquity.common is for things imported by both client and server and this module is only used by the server. Can you move it and its tests over please?